HW01/Vladimir_Sereda/vladimir_sereda_hw_01_t_01.rb

```diff
@@ -0,0 +1,21 @@
+puts 'Number of hourse'
+hours = gets.to_i
+
+puts 'Number of minutes'
+minutes = gets.to_i
+
+def angel(hours, minutes)
+  # Cloc hand position
+  a = ((hours * 60) + minutes) * 0.5
```

Comment:
`Uncommunicative Name`. Please don't use something like this for the variables (`a`, `b`). It would be better to use the name from the comment above 

```suggestion
  cloc_hand_position = ((hours * 60) + minutes) * 0.5
```